src/parser/csql_grammar.y

```diff
@@ -472,6 +494,19 @@ int g_original_buffer_len;
    DBG_PRINT                                            \
 } while (0)
 
+#define MAKE_MONETARY_LITERAL(rv, iv, b_p, sign_text, currency_type) do {    \
```

Comment:
This macro has no use now. Is it used sometime?

Reply:
Ah, I found some repetitive code and prepared to replace it with this macro.
Since it is not directly related to this issue, it has not been modified, but the defined macro is not deleted and remains as it is.

Reply:
Yep